app/src/main/res/values/strings.xml

```diff
@@ -281,6 +281,7 @@
     <string name="options">Options</string>
     <string name="starred">Starred</string>
     <string name="set_scale">Set scale</string>
+    <string name="set_width_scale">Set width scale</string>
```

Comment:
Let's rename these to:
"Set height"
"Set width"

Makes stuff clearer for screen reader users